### DISCUSSION TITLE
username can't contain dots

### DISCUSSION THREAD

**Discussion by sanojjonas** [milo526/homebridge-tuya-web#578] [Q&A]
this is a very annoying thing because my username is my email address.
also the example shows an email address being used as a username.

<img width="855" alt="image" src="https://github.com/milo526/homebridge-tuya-web/assets/3254187/9013e2b2-5e11-4cb7-9dec-01cfe250d7c1">

**Comment by milo526** [comment]
Feel free to improve the documentation 😄 

Technically I can't fix this; this is a limit that Tuya imposes upon us.
